src/styles/blocks/card.scss

```diff
@@ -0,0 +1,130 @@
+.container {
+  box-sizing: border-box;
+  padding: 50px 40px;
+  display: grid;
+  place-content: center;
+  place-items: center;
+  row-gap: 48px 46px;
```

Comment:
The `row-gap` property is used to set the vertical spacing between rows, but here it seems to have two values which is not valid for `row-gap`. If you intended to set both the row and column gaps, use the `gap` property instead.